python/setuputils.py

```diff
@@ -106,6 +111,8 @@ def use_raft_package(raft_path, cpp_build_path,
         print("-- Using already existing RAFT folder, source located at")
         print(os.path.realpath('cuml/raft'))
 
+    return raft_path + '/cpp/include'
```

Comment:
How about `os.path.join`? I always like that I have to worry less about slash management then.